cudamapper/src/main.cpp

```diff
@@ -180,6 +232,15 @@ int main(int argc, char *argv[])
         query_start = query_end + 1;
         query_end = query_start + index_size;
     }
+
+    // Insert empty overlap vector to denote end of processing.
```

Comment:
What happens if two indices just have no overlaps? This seems like a bad poison pill.